manifests/init.pp

```diff
@@ -45,6 +53,8 @@
   validate_bool($local_lib)
   validate_string($config_template)
   validate_string($package_ensure)
+  validate_string($ftp_proxy)
+  validate_string($http_proxy)
```

Comment:
If the defaults for the new parameters are `undef`, these validations will fail.